README.md

```diff
@@ -24,16 +24,16 @@ Prerequisites
 Docker ...
 
 ```bash
-# Clone this repository into `~/datamapper/`
+# Clone this repository into your workspace
 mkdir -p ~/datamapper && cd ~/datamapper
 git clone git@github.com:firespring/dm-dev.git
 cd dm-dev
 
 # DO THIS First -> Configure variables in dm-dev/.env file
```

Comment:
Is it possible to change the color of this? it's grey like normal comments so I skipped over this part and which caused errors